src/app/helper-families/helper-families.component.ts

```diff
@@ -145,7 +145,7 @@ export class HelperFamiliesComponent implements OnInit {
         fd.floor,
         fd.city],
       from: fd,
-      where: () => [fd.deliverStatus.isEqualTo(DeliveryStatus.ReadyForDelivery).and(fd.courier.isEqualTo(''))]
+      where: () => [fd.deliverStatus.isEqualTo(DeliveryStatus.ReadyForDelivery).and(fd.courier.isEqualTo('')).and(fd.quantity.isLessThan(settings.maxDeliverisQuantityToShowForIndependent))]
```

Comment:
this will now also affect assignments from the dist center admin. this logic should only affect "Indie" helper and relates to the function assignNewDelivery in mlt-families.component.ts
This function opens a dialog of SelectListComponent and there we should filter the list. 
Should also add one more requirement there - the dialog must not allow selection of more than "X" items. 
X will be (3-{number of already assigned})